lib/std/macho.zig

```diff
@@ -268,6 +268,343 @@ pub const nlist_64 = extern struct {
     n_value: u64,
 };
 
+/// Defines the attributes of a LC_TWOLEVEL_HINTS load command
+pub const twolevel_hints_command = struct { 
+    // Set to LC_TWOLEVEL_HINTS
+    cmd: u32,
+
+    // @sizeOf(twolevel_hints_command)
+    cmdsize: u32,
+
+    // offset in file
+    offset: u32,
+
+    // Number of twolevel_hint located at offset
+    nhints: u32
+};
+
+/// Specifies an entry in the two-level namespace hint table
+pub const twolevel_hint = struct { 
+    /// Subimage in which the symbol is defined  
+    /// It is an index into the list of images that make up the umbrella image
+    isub_image: u8, 
+
+    /// Symbol index into the table of contents of the image specified by the isub_image
+    itoc: u24 
+};
+
+/// Defines a variable length string
+pub const lc_str = union {
+    /// Byte offset from start of load command to start of str data
+    offset: u32, 
+
+    /// A pointer to an array of bytes / Not used in Macho-O files
+    ptr: *u8
+};
+
+/// Defines data used by the dynamic linker to match a shared 
+/// library against the files that have linked to it
+pub const dylib = struct { 
+    /// Name of the shared library
+    name: lc_str, 
+
+    /// Date and time when the shared library was built
+    timestamp: u32, 
+
+    /// Current version of the shared library
+    current_version: u32, 
+
+    /// Compatibility version of the shared library
+    compatibility_version: u32
+};
+
+/// Defines the attributes of the LC_LOAD_DYLIB and LC_ID_DYLIB load command
+pub const dylib_command = struct { 
+    /// Either LC_LOAD_DYLIB or LC_LOAD_WEAK_DYLIB or LC_ID_DYLIB
```

Comment:
Should this be an enum?:
```zig
pub const DylibCommand = extern enum(u32) {
  LOAD_DYLIB,
  LOAD_WEAK_DYLIB,
  ID_DYLIB,
}
```

similar for other cmd-like fields in this file.

Reply:
Most of these are already defined as const variables so I believe not, however we could move them into an enum if necessary